test/apiv2/10-images.at

```diff
@@ -210,6 +210,18 @@ t POST "images/load" ${TMPD}/test.tar 200 \
 t GET libpod/images/quay.io/libpod/alpine:latest/exists  204
 t GET libpod/images/quay.io/libpod/busybox:latest/exists  204
 
+CONTAINERFILE_WITH_ERR_TAR="${TMPD}/containerfile.tar"
+cat > $TMPD/containerfile << EOF
+FROM quay.io/fedora/fedora
```

Comment:
Is there a critical reason for using fedora here? Can you use `$IMAGE` instead? It's smaller, and already cached anyway. Fedora image is huge, which will slow down tests and will also introduce network flakes.